pkg/ui/server.go

```diff
@@ -48,8 +48,13 @@ func NewServer(ctx context.Context, uiCfg *config.UIServiceConfig, p *justificat
 		return nil, fmt.Errorf("failed to create renderer: %w", err)
 	}
 
+	uic, cerr := controller.New(h, p, uiCfg.Allowlist, ctx)
+	if cerr != nil {
+		return nil, fmt.Errorf("failed to create controller: %w", cerr)
+	}
```

Comment:
No need to use a different err name here.
```suggestion
	uic, err := controller.New(h, p, uiCfg.Allowlist, ctx)
	if err != nil {
		return nil, fmt.Errorf("failed to create controller: %w", err)
	}
```

Reply:
Updated in the latest version.